styles.css

```diff
@@ -0,0 +1,100 @@
+body {
+  background-image: url(/images/Portfolio\ background\ shapes\ mobile.png);
+  background-size: cover;
+  background-color: #191765;
+  color: #f5f5ff;
+  box-sizing: border-box;
+  font-family: sans-serif, Roboto;
+  font-style: normal;
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  padding: 20px 20px;
+}
+
+/*
+   Header
+*/
+.header-box {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header-box p {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 16px;
+}
+
+/*
+  TOOLS & SKILLS
+*/
+
+.name-info {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.name-info h1 {
+  color: #7d73fc;
+  font-size: 35px;
+  text-align: center;
+}
+
+.name-info h2 {
+  font-size: 30px;
+  text-align: center;
+}
+
+.box-tools {
+  display: flex;
+  flex-direction: column;
+}
+
+.box-tools > * {
+  padding: 10px 0;
+}
+
+#intro {
+  text-align: center;
+  font-size: 20px;
+}
```

Comment:
- [ ] Consider adding the required properties to this `intro` so as to match your chosen template.

Figma design:
<img width="135" alt="Loom_MB1AZIRaFF" src="https://user-images.githubusercontent.com/81468675/148381040-45cafc23-b7ba-4ced-93f5-00f9fa4adaaa.png">